.github/workflows/makefile.yml

```diff
@@ -34,7 +37,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        arch: [ amd64 ]
         rust: [ stable ]
```

Comment:
why use `stable` rust on `unit Test` on [amd64], it that useful?

Reply:
> why use `stable` rust on `unit Test` on [amd64], it that useful?

```
    container:
      image: ${{ matrix.arch }}/rust
```
Add this variable, which can be used to specify the image

Reply:
`stable` used to be useful, but now we specify the nightly version, this configuration seems useless